gadget/install/install.go

```diff
@@ -157,6 +157,7 @@ func Run(model gadget.Model, gadgetRoot, kernelRoot, bootDevice string, options
 		return role == gadget.SystemData || role == gadget.SystemSave
 	}
 	var keyForRole map[string]keys.EncryptionKey
+	var devicesForRoles = map[string]string{}
```

Comment:
Nitpick: 
```suggestion
	var devicesForRoles map[string]string
```
or
```suggestion
	devicesForRoles := map[string]string{}
```

Reply:
the first suggestion is not equivalent because that's a nil map that will panic if written to